src/containers/shared/components/Transaction/XChainAccountCreateCommit/parser.ts

```diff
@@ -0,0 +1,20 @@
+import formatAmount from '../../../../../rippled/lib/txSummary/formatAmount'
+
+export function parser(tx: any, meta: any) {
+  const affectedNodes = meta.AffectedNodes
+  const bridgeMeta = affectedNodes.filter(
+    (node: any) =>
+      node.ModifiedNode && node.ModifiedNode.LedgerEntryType === 'Bridge',
+  )[0]
+  // TODO: somehow get the bridge owner via ledger_entry
+  // AffectedNodes won't contain the bridge owner if the transaction fails
+  return {
```

Comment:
I was thinking it would be nice to make the parser async so that additional info could be looked up.